Makefile

```diff
@@ -134,7 +134,7 @@ generate: controller-gen
 # Generate manifests e.g. CRD, RBAC etc.
 manifests: controller-gen
 	@echo Updating generated manifests
-	$(CONTROLLER_GEN) rbac:roleName=manager-role crd:generateEmbeddedObjectMeta=true paths=./api/... webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+	$(CONTROLLER_GEN) rbac:roleName=manager-role crd:generateEmbeddedObjectMeta=true,allowDangerousTypes=true paths=./api/... webhook paths="./..." output:crd:artifacts:config=config/crd/bases
```

Comment:
What are the dangerous types?

Reply:
`/Users/mparida/go/pkg/mod/github.com/rook/rook/pkg/apis@v0.0.0-20231226081419-213dfaadde70/ceph.rook.io/v1/types.go:978:18: found float, the usage of which is highly discouraged, as support for them varies across languages. Please consider serializing your float as string instead. If you are really sure you want to use them, re-run with crd:allowDangerousTypes=true`

This is the source of the warning https://github.com/rook/rook/blob/c3b479727cb5b2fef26bf2ca9a60ab006b4c74a1/pkg/apis/ceph.rook.io/v1/types.go#L995